app/lib/efile/md/md502_calculator.rb

```diff
@@ -16,6 +16,12 @@ def initialize(year:, intake:, include_source: false)
           lines: @lines,
           intake: @intake
         )
+
+        @md502cr = Efile::Md::Md502crCalculator.new(
+          value_access_tracker: @value_access_tracker,
+          lines: @lines,
+          intake: @intake
+        )
       end
```

Comment:
sorry was lurking because i was trying to do the same thing on my PR but i think this needs to be Efile::Md::Md502CrCalculator.new

Reply:
ah no i named the file `md502cr_calculator` which means the cr is treated like part of the same word as Md502. we kind of had two different conventions going on already (`md502b`, `md502_su`) so i just picked one arbitrarily